python-packages/securesync/engine/__init__.py

```diff
@@ -1,352 +0,0 @@
-"""
```

Comment:
This is likely also going to break imports on the ka-lite-central repo. We'll need to fix things up there too.